test/e2e/infinispan/smoke_test.go

```diff
@@ -65,7 +65,7 @@ func TestBaseFunctionality(t *testing.T) {
 	verifyNoPVCs(assert, ispn)
 	verifyLabelsAndAnnotations(assert, require, ispn)
 	verifyDefaultAuthention(require, ispn)
-	verifyScheduling(assert, require, ispn)
+	verifyScheduling(assert, require, ispn, 30)
```

Comment:
To reflect the changes suggested in this review, we should set an explicit value in the Infinispan CR that is not equal to the default and assert that it has been set.